server/env.ts

```diff
@@ -611,6 +611,29 @@ export class Environment {
   @IsOptional()
   public AWS_S3_ACL = process.env.AWS_S3_ACL ?? "private";
 
+  /**
+   * Which file storage system to use
+   */
+  public FILE_STORAGE = process.env.FILE_STORAGE;
```

Comment:
Add `@IsOptional()` validation

Reply:
And also default it to `s3` I suppose?

Reply:
I suppose we have to for now, in that case IsOptional not needed